index.d.ts

```diff
@@ -192,6 +192,26 @@ export interface StringifyOptions {
 	```
 	*/
 	readonly sort?: ((itemLeft: string, itemRight: string) => number) | false;
+
+	/**
+	Skip the key in stringify result if value is `null` or `undefined`.
```

Comment:
This makes it sound like it will **not** skip `undefined` by default, which it does.